src/components/appExit/DockerContainerShutdown.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react';
+import { ipcRenderer } from 'electron';
+import { useStoreActions } from 'store';
+import { useAsyncCallback } from 'react-async-hook';
+import { Modal, Spin } from 'antd';
+import { LoadingOutlined } from '@ant-design/icons';
+import { usePrefixedTranslation } from 'hooks';
+
+const DockerContainerShutdown: React.FC = () => {
```

Comment:
I think it would be better to move this component into the `common` dir. It's not very useful to have a new dir with just one file in it.